### PR TITLE
updated citations

### PR DESCRIPTION
**Pull request by mclark58**
updated citations to PLOS format, increased patch number by 1, and created RELEASE_NOTES.md file

### CODE REVIEW

File: ui/narrative/methods/run_QUAST_app/display.yaml [hunk 1]
```diff
@@ -30,19 +30,19 @@ parameters :
...
+    <a href="http://quast.sourceforge.net/" target="_blank">QUAST</a>
+    assembly quality assessment tool.
     </p>
 
-publications: 
+publications:
     -
         display-text: |
-            Alexey Gurevich, Vladislav Saveliev, Nikolay Vyahhi and Glenn Tesler, QUAST: quality assessment tool for genome assemblies, Bioinformatics (2013) 29 (8): 1072-1075. doi: 10.1093/bioinformatics/btt086 First published online: February 19, 2013
+            Gurevich A, Saveliev V, Vyahhi N, Tesler G. QUAST: quality assessment tool for genome assemblies. Bioinformatics. 2013;29: 1072–1075. doi:10.1093/bioinformatics/btt086
```
Comment (MrCreosote) [inline ui/narrative/methods/run_QUAST_app/display.yaml]: Is the issue # left out intentionally?

File: ui/narrative/methods/run_QUAST_app/display.yaml [hunk 2]
```diff
@@ -30,19 +30,19 @@ parameters :
...
         link: https://www.ncbi.nlm.nih.gov/pubmed/23422339
         pmid: 23422339
     -
         display-text: |
-            Alla Mikheenko, Gleb Valin, Andrey Prjibelski, Vladislav Saveliev, Alexey Gurevich, Icarus: visualizer for de novo assembly evaluation, Bioinformatics (2016). doi: 10.1093/bioinformatics/btw379 First published online: July 4, 2016
+            Mikheenko A, Valin G, Prjibelski A, Saveliev V, Gurevich A. Icarus: visualizer for de novo assembly evaluation. Bioinformatics. 2016;32: 3321–3323. doi:10.1093/bioinformatics/btw379
```
Comment (MrCreosote) [inline ui/narrative/methods/run_QUAST_app/display.yaml]: Same ?

File: ui/narrative/methods/run_QUAST_app/display.yaml (hunk 1 above)
Comment (mclark58) [reply ui/narrative/methods/run_QUAST_app/display.yaml]: That's just the way Zotero generates the Public Library of Science citations. Going back through my other edited ones, it doesn't seem to have included issue numbers there either.

File: ui/narrative/methods/run_QUAST_app/display.yaml (hunk 2 above)
Comment (mclark58) [reply ui/narrative/methods/run_QUAST_app/display.yaml]: ^^^

**Pull request merged**